Okta.Xamarin/Okta.Xamarin.UITest.Shared/TestOktaConfig.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
```

Comment:
This can be probably managed in a diff PR, but all files should include the Okta header.

Reply:
I'll add follow up item to address this along with formatting.